spec/index.md

```diff
@@ -40,7 +40,7 @@ enables the app to protect secrets.   Pure client-side apps
 (for example, HTML5/JS browser-based apps, iOS mobile
 apps, or Windows desktop apps) can provide adequate security, but they may be unable to "keep a secret" in the OAuth2 sense.  In other words, any "secret" key, code, or
 string that is statically embedded in the app can potentially be extracted by an end-user
-or attacker. Hence security for these apps can't depend on secrets embedded at
+or attacker. Hence security for these apps can not depend on secrets embedded at
```

Comment:
I'm going to be totally pedantic and ask for `cannot` here, to avoid confusion with "might not".